### PR TITLE
add EXAMPLES for subcommands of `issue` subcommand

### PR DESCRIPTION
**Pull request by maxice8**
**Types of changes**

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation
- [ ] Chore (Related to CI or Packaging to platforms)

### CODE REVIEW

File: commands/issue/unsubscribe/issue_unsubscribe.go [hunk 1]
```diff
@@ -18,7 +19,11 @@ func NewCmdUnsubscribe(f *cmdutils.Factory) *cobra.Command {
 		Short:   `Unsubscribe to an issue`,
 		Long:    ``,
 		Aliases: []string{"unsub"},
-		Args:    cobra.ExactArgs(1),
+		Example: heredoc.Doc(`
+			$ glab unsubscribe 123
+			$ glab unsub 123
```
Comment (profclems) [inline commands/issue/unsubscribe/issue_unsubscribe.go]:
```suggestion
			$ glab issue unsubscribe 123
			$ glab issue unsub 123
```

File: commands/issue/reopen/issue_reopen.go [hunk 2]
```diff
@@ -19,7 +20,11 @@ func NewCmdReopen(f *cmdutils.Factory) *cobra.Command {
 		Short:   `Reopen a closed issue`,
 		Long:    ``,
 		Aliases: []string{"open"},
-		Args:    cobra.ExactArgs(1),
+		Example: heredoc.Doc(`
+			$ glab reopen 123
+			$ glab open 123
```
Comment (profclems) [inline commands/issue/reopen/issue_reopen.go]:
```suggestion
			$ glab issue reopen 123
			$ glab issue open 123
```

**Review by profclems — CHANGES_REQUESTED** [review]

File: commands/issue/subscribe/issue_subscribe.go [hunk 3]
```diff
@@ -18,7 +19,11 @@ func NewCmdSubscribe(f *cmdutils.Factory) *cobra.Command {
 		Short:   `Subscribe to an issue`,
 		Long:    ``,
 		Aliases: []string{"sub"},
-		Args:    cobra.ExactArgs(1),
+		Example: heredoc.Doc(`
+			$ glab subscribe 123
+			$ glab sub 123
```
Comment (profclems) [inline commands/issue/subscribe/issue_subscribe.go]:
```suggestion
			$ glab issue subscribe 123
			$ glab issue sub 123
```

**Review by profclems — APPROVED** [review]

**Comment by maxice8** [comment]
Should this wait and be adapted for #546 ?

**Comment by profclems** [comment]
Yeah... We'd have to add examples specifying URLs as args.

**Comment by profclems** [comment]
This can now be adapted for #546

**Comment by maxice8** [comment]
Thanks, I'll adapt tomorrow, tonight is :video_game: night.

**Comment by maxice8** [comment]
rebased and wrote examples with URLs, can you check ?

**Pull request merged**